schedule.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+permalink: /schedule/
+title: Schedule
+---
+
+
+{::options parse_block_html="true" /}
+<div class="schedule">
+<section>
+
+| Talk | Speaker | Time |
+| ------------- | ------------- |
+| TODO: Contributing to Rust | Manish Goregaokar | 9:00-9:30 |
+| The Illustrated Adventure Survival Guide for New Rustaceans | Liz Baillie | 9:40-9:55 |
+| Taking Rust to Production: Lessons Learned From the Habitat Project | Fletcher Nichol | 10:05-10:35 |
+| Break | | 10:35-10:50 |
+| Building SSH servers in minutes | Pierre-Étienne Meunier | 10:50-11:20 |
+| How I Convinced the World's Largest Package Manager to Use Rust, and So Can You! | Ashley Williams | 11:30-12:00 |
+| Writing Idiomatic Libraries in Rust | Pascal Hertleif | 12:10-12:40 |
+| Lunch | | 12:40-1:40 |
+| Making a game in Rust | lislis | 1:40-1:55 |
+| GStreamer & Rust – A perfect match | Sebastian Dröge & Luis de Bethencourt | 2:05-2:35 |
+| Sōzu, a hot reconfigurable reverse HTTP proxy | Geoffroy Couprie | 2:45-3:25 |
+| Break | | 3:25-3:40 |
+| Interaction with real-world JavaScript from compiled Rust | Ingvar Stepanyan | 3:40 - 4:10  |
+| vagga -- containerization without daemons | Paul Colomiets | 4:20 - 5:00 |
+| TODO: Automation in Rust | Emily Dunham | 5:15-6:00 |
```

Comment:
We will have 15 minutes between vagga and Emily's talk.

I will be merging this **Saturday Morning** if there are no objections.